### PR TITLE
docs: rewrite `guides/sidebar` examples to be more generic

### PR DESCRIPTION
**Pull request by HiDeoo**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

While working #1750, I realized that the "Sidebar Navigation" guide was present in a lot of search results for unrelated queries (and will be even more after the `pagefind` v1.1.0 update). This is because the sidebar examples embedded in the guide are re-using the Starlight documentation site's sidebar making it appear in search results for queries like "component", "CSS", etc.

This pull request rewrites the guide examples to be more generic and use constellations/stars/moons instead that are being used in other guides.

- Previous example links were actually linking to said pages, now they do not redirect anywhere (this uses an invalid anchor - `#` scroll to the top of the page and others solutions involve JS)
- The "Internationalization" section involves Brazilian Portuguese translations, I linked the reference I used for the translations but this should be reviewed by a native speaker

### CODE REVIEW

File: docs/src/content/docs/guides/sidebar.mdx [hunk 1]
```diff
@@ -393,24 +389,24 @@ The `label` property will be used for the default locale and for languages witho
 starlight({
 	sidebar: [
 		{
-			label: 'Guides',
+			label: 'Constellations',
 			translations: {
-				'pt-BR': 'Guias',
+				'pt-BR': 'Constelação',
```
Comment (HiDeoo) [inline docs/src/content/docs/guides/sidebar.mdx:394]: Reference: https://pt.wikipedia.org/wiki/Constela%C3%A7%C3%A3o

File: docs/src/content/docs/guides/sidebar.mdx [hunk 2]
```diff
@@ -393,24 +389,24 @@ The `label` property will be used for the default locale and for languages witho
...
 			},
 			items: [
 				{
-					label: 'Components',
+					label: 'Andromeda',
 					translations: {
-						'pt-BR': 'Componentes',
+						'pt-BR': 'Andrômeda',
```
Comment (HiDeoo) [inline docs/src/content/docs/guides/sidebar.mdx:400]: Reference: https://pt.wikipedia.org/wiki/Andr%C3%B4meda_(constela%C3%A7%C3%A3o)

File: docs/src/content/docs/guides/sidebar.mdx [hunk 3]
```diff
@@ -393,24 +389,24 @@ The `label` property will be used for the default locale and for languages witho
...
 					},
-					link: '/guides/components/',
+					link: '/constellations/andromeda/',
 				},
 				{
-					label: 'Internationalization (i18n)',
+					label: 'Scorpius',
 					translations: {
-						'pt-BR': 'Internacionalização (i18n)',
+						'pt-BR': 'Escorpião',
```
Comment (HiDeoo) [inline docs/src/content/docs/guides/sidebar.mdx:407]: Reference: https://pt.wikipedia.org/wiki/Scorpius

**Review by delucis — APPROVED** [review]
This is absolutely fantastic — thank you @HiDeoo!

I wonder if @Fryuni or @yanthomasdev are available to confirm the Portuguese snippet: https://starlight-git-fork-hideoo-hd-docs-refactor-601622-astrodotbuild.vercel.app/guides/sidebar/#internationalization

File: docs/src/content/docs/guides/sidebar.mdx (hunk 1 above)
Comment (yanthomasdev) [inline docs/src/content/docs/guides/sidebar.mdx:394]: "Constelação" is the singular form, I suppose you wanted the plural form, which is "Constelações"
```suggestion
				'pt-BR': 'Constelações',
```

Comment (HiDeoo) [reply docs/src/content/docs/guides/sidebar.mdx]: You are 100% correct, thanks :raised_hands:

**Pull request merged**